src/contributors/yusufalp.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Yusuf Alp", 
+  "desc": "Front end web developer who enjoys puzzles and coding.",
+  "github" : "https://github.com/yusufalp",
+  "twitter" : "https://twitter.com/yalp122", 
```

```suggestion
  "twitter": "https://twitter.com/yalp122", 
```
Remove unwanted `white-space`.